### PR TITLE
adding tomcat7

### PR DESCRIPTION
**Pull request by hsachinraj**
Adding tomcat7 artifact

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@hsachinraj__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: Artifacts/linux-openjdk-tomcat7/Artifactfile.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+    "title": "Tomcat 7",
+    "description": "Install TRomcat 7 and OpenJDK on Linux",
```
Comment (drwill-ms) [inline Artifacts/linux-openjdk-tomcat7/Artifactfile.json]: Tomcat misspelled

File: Artifacts/linux-openjdk-tomcat7/Artifactfile.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "tags": [
+        "Tomcat",
+		"Linux"
```
Comment (drwill-ms) [inline Artifacts/linux-openjdk-tomcat7/Artifactfile.json]: tab -> spaces

File: Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+sudo apt-get -y update
+sudo apt-get install -y openjdk-8-jdk
+sudo apt-get -y update --fix-missing
+sudo apt-get install -y openjdk-8-jdk
```
Comment (drwill-ms) [inline Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh]: Not all Linux distros have apt-get. In other artifacts we've added conditional support for apt-get or yum. Here's an example: https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/linux-install-mongodb/linux_install_mongodb_v1.sh

File: Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#edit the tomcat users file
+sudo apt-get install -y tomcat7 tomcat7-admin
+sed -i "s#</tomcat-users>##g" /etc/tomcat7/tomcat-users.xml; \
+	echo '  <role rolename="manager-gui"/>' >>  /etc/tomcat7/tomcat-users.xml; \
+	echo '  <role rolename="manager-script"/>' >>  /etc/tomcat7/tomcat-users.xml; \
+	echo '  <role rolename="manager-jmx"/>' >>  /etc/tomcat7/tomcat-users.xml; \
+	echo '  <role rolename="manager-status"/>' >>  /etc/tomcat7/tomcat-users.xml; \
+	echo '  <role rolename="admin-gui"/>' >>  /etc/tomcat7/tomcat-users.xml; \
+	echo '  <role rolename="admin-script"/>' >>  /etc/tomcat7/tomcat-users.xml; \
+	echo '  <user username="vmadmin" password="P2ssw0rd" roles="manager-gui, manager-script, manager-jmx, manager-status, admin-gui, admin-script"/>' >>  /etc/tomcat7/tomcat-users.xml; \
```
Comment (drwill-ms) [inline Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh]: Should the password be a parameter from the artifact?

File: Artifacts/linux-openjdk-tomcat7/Artifactfile.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    ],
+    "iconUri": "https://raw.githubusercontent.com/hsachinraj/devtestlabrepo/master/tomcat.png",
```
Comment (drwill-ms) [inline Artifacts/linux-openjdk-tomcat7/Artifactfile.json]: icon url will need updating to our master branch

**Comment by drwill-ms** [comment]
Sorry about the delay on your PR @hsachinraj. I'll get some of our more Linux-savvy team members to take a look.

Can you speak to the testing you've done with this artifact? Which Linux gallery images have you applied it to?

File: Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh [hunk 6]
```diff
@@ -0,0 +1,20 @@
+sudo apt-get -y update
```
Comment (ryanvog) [inline Artifacts/linux-openjdk-tomcat7/linux_install_tomcat7.sh]: In order to be considered for inclusion in the public artifact repository, you will to support all variants of linux supported on Azure: Debian Ubuntu, CoreOS, CentOS, SuSE, and Fedora RedHat.  What this means in practical terms is you will need to do the work to determine which package manage is present.  See other public linux artifacts for examples of how to do this.

**Comment by mmydland** [comment]
This hasn't been touched in 8 months so I am closing it down

**Pull request closed without merging**